api_db.php

```diff
@@ -70,7 +70,10 @@ function SQLite3_connect($trytoreconnect)
 	{
 		$from = intval($_GET["from"]);
 		$until = intval($_GET["until"]);
-		$results = $db->query('SELECT timestamp,type,domain,client,status FROM queries WHERE timestamp >= '.$from.' AND timestamp <= '.$until.' ORDER BY timestamp ASC');
+		$stmt = $db->prepare("SELECT timestamp, type, domain, client, status FROM queries WHERE timestamp >= :from AND timestamp <= :until ORDER BY timestamp ASC");
+		$stmt->bindValue(":from", $from);
```

Comment:
I don't really get the point of doing this here. We already have
```
$from = intval($_GET["from"]);
$until = intval($_GET["until"]);
```
so there cannot be any e.g. code injection as everything that is not an integer will be discarded by the above mentioned subroutines. Here is [the definition of `intval()`](http://php.net/manual/en/function.intval.php):
```
int intval ( mixed $var [, int $base = 10 ] )
```
> Returns the integer value of var, using the specified base for the conversion (the default is base 10). intval() should not be used on objects, as doing so will emit an E_NOTICE level error and return 1.

Reply:
Mainly for consistency.